osu.Game/Overlays/MusicController.cs

```diff
@@ -219,6 +219,13 @@ private void load(OsuGameBase game, OsuColour colours, LocalisationEngine locali
             playlist.StateChanged += s => playlistButton.FadeColour(s == Visibility.Visible ? colours.Yellow : Color4.White, 200, Easing.OutQuint);
         }
 
+        private bool? conditionalSeek(double progress)
+        {
+            if (current.Track.Looping)
```

Comment:
It is bad to make the assumption that "looping" means "not playing".

There should probably instead be an `AllowUserSeek` in `OsuScreen` (see `AllowBeatmapRulesetChange` for an example).

Reply:
Yeah, that seems preferable.
 
I am however struggling to find how the MusicController has access to OsuScreen? MusicController doesn't seem to have any connection to the currently active screen? Do you want me to implement that?

Reply:
You would need to pass it through `OsuGame` most likely. You could copy the method being used by `AllowBeatmapRulesetChange` (ie. just set a bool on `MusicController` every `UpdateAfterChildren`). Another possibility would be using a bindable and binding to the current screen in `OsuGame` (when the new screen is pushed) but it may be best to follow suit for now.